docs/docs/manual/installing.md

```diff
@@ -7,53 +7,269 @@ sidebar_label: Installing
+
+
+#### Unstable versions
+
+If you need a recently developed function, and are willing to risk some untested code, you can look at [the most recent items in the reverse-chronological list](https://github.com/OpenRefine/OpenRefine/releases) and see what changes appeal to you. 
+
+“Beta” and “release candidate” versions may both have unreported bugs and are most suitable for people who are wiling to help us troubleshoot these versions by [creating bug reports](https://github.com/OpenRefine/OpenRefine/issues).  
+
+For the absolute latest development updates, see the [snapshot releases](https://github.com/OpenRefine/OpenRefine-nightly-releases/releases). These are created with every commit. 
+
+
+#### What’s Changed 
+
+Our [latest release is at the time of writing is OpenRefine 3.4](link goes here!), released XXXX XX 2020. The major changes in this version are listed on the [3.4 final release page](link goes here!) with the downloadable packages. 
+
+
+
+:::note
+
+[This link always goes to the actual latest release](https://github.com/OpenRefine/OpenRefine/releases/latest), whether this user manual is updated or not. 
+
+:::
 
+You can find information about all of our releases on the [Releases page on Github](https://github.com/OpenRefine/OpenRefine/releases).
 
-*   What’s Changed (a plain language summary of major improvements only)
-*   Layman explainer of beta, RC, and final
-    *   Beta
-    *   Development version
-        *   [Nightly releases](https://github.com/OpenRefine/OpenRefine-nightly-releases/releases)
-*   [Changelogs feed](https://github.com/OpenRefine/OpenRefine/releases)
-*   Other distributions
+
+:::tip Other distributions
+
+Our community of contributors may have created a version of OpenRefine that works for you. Look at the list of “Other Distributions” on our [Download page]([https://openrefine.org/download.html](https://openrefine.org/download.html)).
+
+:::
 
 
 ## Installing or upgrading
 
 
 ### Back up your data
 
+If you are upgrading from an older version of OpenRefine and have projects already on your computer, you should create backups of those projects before you install a new version.
 
+First, [locate your workspace directory](#where-is-data-stored). Then copy everything you find there and paste it into a folder elsewhere on your computer. 
 
-*   Windows
-*   Mac
-*   Linux
+For extra security you can [export your existing OpenRefine projects](exporting.md#export-a-project). 
 
 
-### Install/Upgrade
+:::caution
 
+Take note of the [extensions](#installing-extensions) you have currently installed. They may not be compatible with the upgraded version of OpenRefine. Installations can be installed in two places, so be sure to check both your workspace directory and the existing installation directory. 
```

Comment:
Can you add the link for where-is-data-stored for the "workspace directory" in this tip?

Reply:
Definitely!